src/remote/remote.h

```diff
@@ -192,23 +193,42 @@ struct Rdb : public Firebird::GlobalStorage, public TypedHandle<rem_type_rdb>
 };
 
 
+const SQUAD NULL_BLOB = {0, 0};
+
+inline bool operator==(const SQUAD& s1, const SQUAD& s2)
```

Comment:
I think these operators should better be defined in the same header as the type.

Reply:
`SQUAD` is defined in _src\include\fb_types.h_, I thought it is public header, but it seems not.
So, if it is OK to move these operators into _src\include\fb_types.h_ - I'll do it.

Reply:
Done